### ISSUE TITLE
Debian 9: packages are broken

### ISSUE THREAD

**Issue by benhartwich** [Icinga/icinga2#7523]
Hi,

it seems that the latest published icinga2 packages for debian 9 are broken: icinga2 (2.11.0-2.stretch)

The following errors occur:

```
root@mail ~ # apt install icinga2
Paketlisten werden gelesen... Fertig
Abhängigkeitsbaum wird aufgebaut.
Statusinformationen werden eingelesen.... Fertig
Einige Pakete konnten nicht installiert werden. Das kann bedeuten, dass
Sie eine unmögliche Situation angefordert haben oder, wenn Sie die
Unstable-Distribution verwenden, dass einige erforderliche Pakete noch
nicht erstellt wurden oder Incoming noch nicht verlassen haben.
Die folgenden Informationen helfen Ihnen vielleicht, die Situation zu lösen:

Die folgenden Pakete haben unerfüllte Abhängigkeiten:
 icinga2 : Hängt ab von: icinga2-bin (= 2.11.0-2.stretch) soll aber nicht installiert werden
E: Probleme können nicht korrigiert werden, Sie haben zurückgehaltene defekte Pakete.
root@mail ~ # apt-get install icinga2-bin
Paketlisten werden gelesen... Fertig
Abhängigkeitsbaum wird aufgebaut.
Statusinformationen werden eingelesen.... Fertig
Einige Pakete konnten nicht installiert werden. Das kann bedeuten, dass
Sie eine unmögliche Situation angefordert haben oder, wenn Sie die
Unstable-Distribution verwenden, dass einige erforderliche Pakete noch
nicht erstellt wurden oder Incoming noch nicht verlassen haben.
Die folgenden Informationen helfen Ihnen vielleicht, die Situation zu lösen:

Die folgenden Pakete haben unerfüllte Abhängigkeiten:
 icinga2-bin : Hängt ab von: libboost-context1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-coroutine1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-date-time1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-filesystem1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-program-options1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-regex1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-system1.67.0 ist aber nicht installierbar
               Hängt ab von: libboost-thread1.67.0 ist aber nicht installierbar
E: Probleme können nicht korrigiert werden, Sie haben zurückgehaltene defekte Pakete.
```
It happened when I upgraded from 2.11.0-1 to 2.11.0-2. I had to uninstall icinga2, because the update crashed my whole installation.

**Comment by dnsmichi** [comment]
https://icinga.com/docs/icinga2/latest/doc/16-upgrading-icinga-2/#added-boost-166

**Comment by benhartwich** [comment]
Thanks, regarding your documentation the official debian stretch repository of icinga should solve the problem? I´m still using this official repository.

**Comment by dnsmichi** [comment]
> On platforms where EPEL or Backports cannot satisfy this dependency

TL;DR - use Debian Backports on Stretch.
